src/services/create-project.service.js

```diff
@@ -11,6 +11,9 @@ import type { ProjectType } from '../types';
 
 const fs = window.require('fs');
 const childProcess = window.require('child_process');
+const path = window.require('path');
+const os = window.require('os');
+const isWin = /^win/.test(os.platform());
```

Comment:
I think it'd be great to add a `services/platform.services.js` file that would export:

- `isWin`, this quick check
- `getWindowsHomeDir`, mentioned above
- `formatCommandForPlatform` also mentioned above.

This is super non-urgent, I'm happy to go in after this merges and do it myself :) just mentioning it in case you're up for it (and so that I don't forget)

Reply:
I moved `isWin` and `getWindowsDir` to `platform.services.js`. I was going to also propose a complete `formatCommad` service but I thought it's not really just for windows support so I think it should be in a separate PR. Using a global service function to get the command can benefit from future library additions like next. Doesn't require to change things on different files for example.